pkg/gds/mock.go

```diff
@@ -88,7 +88,7 @@ func MockConfig() config.Config {
 			CookieDomain: "admin.gds.dev",
 			Audience:     "http://api.admin.gds.dev",
 			Oauth: config.OauthConfig{
-				GoogleAudience:         "4284607864536-notarealgoogleaudience.apps.googleusercontent.com",
+				GoogleAudience:         "http://localhost",
```

Comment:
Changed to make the integration testing easier?

Reply:
The other option was to change the secrets mock to use the fake google audience above, for testing purposes they just have to match so I went for the simpler option.